### PR TITLE
Create README.rst

### PR DESCRIPTION
**Pull request by JordanP**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Scality_ RING installation.
+
+.. _OpenStack: http://openstack.org
+.. _Glance: http://docs.openstack.org/developer/glance/
+.. _Scality: http://scality.com
+
+Installation
+------------
+This package depends on `Scality Sproxyd client`_, a Python client library for Scality Sproxyd connector. It must 
+be installed first before installing this package.
```
Comment (NicolasT) [inline README.rst]: Remove `first`.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+2. Configure Glance API. Edit your *glance-api.conf* file to add the Scality backend to the list of known
+   Glance stores:
+
+   .. code-block:: ini
+
+    [glance_store]
+    stores = glance.store.filesystem.Store,
+             glance.store.scality.Store,
+             glance.store.http.Store
+   **N.B** This is an configuration example. Just make sure *glance.store.scality.Store* appears in the list
```
Comment (NicolasT) [inline README.rst]: `a`, not `an configuration`.

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   of *stores* in the *[glance_store]* section of *glance-api.conf*.
+
+3. Set to which Sproxyd connectors to connect to. Edit your *glance-api.conf* file and add, in the
```
Comment (NicolasT) [inline README.rst]: I'd rephrase the first sentence, e.g.

```
Configure the Sproxyd connectors to use.
```

Comment (NicolasT) [inline README.rst]: Remove 'your' and 'file'. Move the section information to the end of the sentence so there's no sub-sentence.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   *[glance_store]* section, the list of your Sproxyd connectors configured to accept **query by path**:
+
+   .. code-block:: ini
+
+    [glance_store]
+    scality_sproxyd_endpoints = http://4.5.9.2:81/proxy/chord_path/,http://4.5.9.4:81/proxy/arc_path/
+
+4. (optionnal) Use the Scality Store as the default image store. Newly uploaded Glance image will be stored in
+   Scality Ring. As of OpenStack Kilo, the default store has to be set in both the DEFAULT
+   section and the glance_store section, for compatibility reason.
```
Comment (NicolasT) [inline README.rst]: Use special formatting for `DEFAULT` and `glance_store`.

Comment (NicolasT) [inline README.rst]: reasons

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+3. Set to which Sproxyd connectors to connect to. Edit your *glance-api.conf* file and add, in the 
+   *[glance_store]* section, the list of your Sproxyd connectors configured to accept **query by path**:
+
+   .. code-block:: ini
+
+    [glance_store]
+    scality_sproxyd_endpoints = http://4.5.9.2:81/proxy/chord_path/,http://4.5.9.4:81/proxy/arc_path/
+
+4. (optionnal) Use the Scality Store as the default image store. Newly uploaded Glance image will be stored in
```
Comment (NicolasT) [inline README.rst]: `optional`

**Comment by NicolasT** [comment]
Some spelling, wording and formatting comments, otherwise LGTM.

**Comment by JordanP** [comment]
Could you take a final look at it ?

**Comment by NicolasT** [comment]
:+1:
